apps/jest-expo-mock-generator/App.js

```diff
@@ -35,6 +43,35 @@ export default class App extends React.Component {
   }
 }
 
+/**
+ * Sends a log message without truncating it.
+ */
+async function _sendRawLogAsync(message, logUrl) {
+  const headers = {
+    'Content-Type': 'application/json',
+    Connection: 'keep-alive',
+    'Proxy-Connection': 'keep-alive',
+    Accept: 'application/json',
+    'Device-Id': await getInstallationIdAsync(),
+    'Session-Id': new Date().getTime().toString(),
```

Comment:
we should add a comment here to indicate that this is just some placeholder value because session-id does not seem to be used but is required

Reply:
Opted to match the `RemoteLogging` implemetation and pass a real UUID instead, as this was not any harder than explaining this in a comment 😄